Source/NSCell.m

```diff
@@ -1398,10 +1400,10 @@ - (void) setTitleWithMnemonic: (NSString*)aString
   if (r.length > 0)
     {
       NSUInteger location = r.location;
-      
-      
-      [self setTitle: [[aString substringToIndex: location] 
-                        stringByAppendingString: 
+
+
```

Comment:
Since this is being cleaned up, might as well nuke one of the extraneous newlines...